docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,112 @@
+authors: ['josefreitas']
+tags: ['MUI X', 'News']
+---
+
+There's a lot of exciting news in MUI X v6, but there's hardly anything comparable to the revamp we're delivering for the [Date and Time Pickers](https://next.mui.com/x/react-date-pickers/getting-started/).
+
+It's the result of our research and effort to improve these inputs' usability and overall customizability.
+We kept the best aspects of the previous versions, and we're fleshing out the package with several improvements and one fundamental change: a new input type to replace the masked text fields.
+
+## The new date and time fields
+
+This new input type we're talking about is a set of components shortly named [Fields](https://next.mui.com/x/react-date-pickers/fields/).
+They are the base for the new pickers and a revolution in terms of keyboard usability.
+
+In contrast with the masked inputs that only display simple text behind a formatting mask, and were often cumbersome, especially when editing dates, the new fields are smart and specialized in editing date and time numbers.
+It can recognize what's a month, day, or whatever piece of the date you are currently editing and act accordingly.
+
+For example, watch what happens when you edit the month with arrow keys.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/intelligent-field.mp4" type="video/mp4" />
+</video>
+
+As you increase the month, the component automatically watches for valid values throughout each piece of your date input, like you're navigating in a calendar.
+
+The Fields are a perfect match to the calendar view, but they are available as stand-alone components, and you can import them from the latest v6 package.
+
+```jsx
+import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
+
+<DateField label="My first field" />;
+```
+
+They are also included by default on the new pickers.
+
+```jsx
+import { Unstable_NextDatePicker as NextDatePicker } from '@mui/x-date-pickers/NextDatePicker';
+
+<NextDatePicker label="My first v6 picker" />;
+```
+
+## Enhanced keyboard usability
+
+With the new fields, not only can you type the date as a text, but you can use your arrow keys to quickly navigate through each piece of date and time values.
+You can edit them atomically, without, for instance, leaking the year into the month, a common occurrence with masked inputs.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/keyboard-nav.mp4" type="video/mp4" />
+</video>
+
+For a quick comparison, let's check out, for instance, how the two different approaches behave on the most typical scenarios.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/fields-common-usecases.mp4" type="video/mp4" />
+</video>
+
+Try out the live demo of the `DateField` below.
```

Comment:
Not directly related to the blog post, but I believe we should somehow improve the date field to move across the date pieces automatically when there's valid input. Something like [spectrum](https://react-spectrum.adobe.com/react-spectrum/DatePicker.html) does. I'm afraid this we'll be a reason people will stumble when using the component, especially for the first time.